packages/data-context/src/sources/BrowserDataSource.ts

```diff
@@ -76,6 +76,14 @@ export class BrowserDataSource {
     return false
   }
 
+  isVersionSupported (obj: FoundBrowser) {
+    return Boolean(!obj.unsupportedVersion)
+  }
+
+  warning (obj: FoundBrowser) {
```

Comment:
We can remove this, it's not being used since the gql-type is inferring this due to the lack of `resolve`